Backend/TimeTracker.Api/Controllers/ProjectController.cs

```diff
@@ -161,6 +161,43 @@ public async Task<GenericResponseDTO<int>> AddTag(CreateTagDTO newTag)
             };
         }
 
+        /// <summary>
+        /// Adds a user to a project
+        /// </summary>
+        /// <param name="projectInvite">The invite code of the project that the user is added to</param>
+        /// <returns>The Id of the project the user was added to</returns>
+        [Authorize]
+        [HttpPost("AddUserToProject")]
+        public async Task<GenericResponseDTO<int>> AddUserToProject(AddUserToProjectDTO inviteCode)
+        {
+            var currentUserId = authHelper.GetCurrentUserId(User);
+
+            Project project = await database.Projects
+                .FirstOrDefaultAsync(x => x.InviteCode == inviteCode.InviteCode);
+
+            var curUser = await database.Users
+                .FirstOrDefaultAsync(x => x.Id == currentUserId);
```

Comment:
Use `authHelper.GetCurrentUser()` at the top here instead of `authHelper.GetCurrentUserId()` to eliminate the need for this.

Reply:
fixed